package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "starlight",
+  "version": "1.0.0",
+  "description": "Cross-functional skill tracking system for Agile teams.",
+  "main": "index.js",
+  "scripts": {
+    "test": "test-npm",
+    "webpack": "webpack",
+    "webpack:dev": "webpack-dev-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/savoirfairelinux/starlight.git"
+  },
+  "author": "Anthony Laye",
+  "license": "GPLv3",
+  "bugs": {
+    "url": "https://github.com/savoirfairelinux/starlight/issues"
+  },
+  "homepage": "https://github.com/savoirfairelinux/starlight#readme",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.7.0",
+    "css-loader": "^0.28.11",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "^1.1.11",
+    "gulp": "^3.9.1",
+    "less-loader": "^4.1.0",
+    "node-sass": "^4.5.0",
+    "url-loader": "^1.0.1",
+    "webpack": "^3.10.0",
+    "webpack-bundle-tracker": "^0.2.1",
+    "webpack-dev-server": "^2.11.1",
+    "webpack-cli": "1.0.0",
+    "write-file-webpack-plugin": "^4.2.0"
+  },
+  "dependencies": {
+    "less": "^2.3.1",
+    "semantic-ui": "^2.3.1",
+    "bootstrap": "^3.3.7",
```

Comment:
We are using Semantic UI with this project, so no need to install `bootstrap` and `bootstrap-sass`.